modules/external-api/src/main/java/org/opencastproject/external/endpoint/ListProviderEndpoint.java

```diff
@@ -94,26 +93,28 @@ void activate(ComponentContext cc) {
 
   @GET
   @Path("providers.json")
-  @Produces(MediaType.APPLICATION_JSON)
+  @Produces({ ApiMediaType.JSON, ApiMediaType.VERSION_1_10_0 })
   @RestQuery(name = "availableProviders", description = "Provides the list of the available list providers", responses = { @RestResponse(description = "Returns the availables list providers.", responseCode = HttpServletResponse.SC_OK) }, returnDescription = "")
-  public Response getAvailablesProviders(@Context HttpHeaders headers) {
+  public Response getAvailablesProviders(@HeaderParam("Accept") String acceptHeader) {
```

Comment:
Oh I just noticed: `getAvailableSSSSSSSProviders`? 🤔

Reply:
What?

Reply:
The plural s after "available" confuses me. Or am I being dumb rn? 🙈

Reply:
Ah, I see.